hr_language/i18n/fr.po

```diff
@@ -250,6 +251,11 @@ msgid "Languages"
 msgstr "Langues"
 
 #. module: hr_language
+#: help:hr.employee,language_ids:0
+msgid "Languages of this employee"
+msgstr "Languages de cet employé"
```

Comment:
"Langues de cet employé".

"Language" translates to "Langue" in french, as "Langage" (and not "Language") has a different meaning.

So this should be fixed everythere in the french translation.